docs/platform/howto/byoc/create-custom-cloud.md

```diff
@@ -822,6 +879,19 @@ Your new custom cloud is ready to use only after its status changes to
    --account "ACCOUNT_IDENTIFIER"
    ```
 
+   Replace the following:
+
+   - `ORGANIZATION_IDENTIFIER` with the identifier of your Aiven organization that is to be
+     connected with your own cloud account to create the custom cloud,
+     for example `org484338bebd2`. You can extract `ORGANIZATION_IDENTIFIER` from the
+     output of the `avn organization list` command.
+   - `CUSTOM_CLOUD_IDENTIFIER` with the identifier of your custom cloud, which you can
+     extract from the output of the `avn byoc create` command, for example
+     `018b6442-c602-42bc-b63d-438026133f60`.
+   - `ACCOUNT_IDENTIFIER` with the identifier of your account (organizational unit) in Aiven,
+     for example `a484338c34d7`. You can extract `ACCOUNT_IDENTIFIER` from the output of
+     the `avn account list` command.
```

Comment:
`avn organization list`'s output provides the same details is a more concise way.

```
avn organization list
ORGANIZATION_NAME  ORGANIZATION_ID  ACCOUNT_ID    CREATE_TIME           UPDATE_TIME           TIER    
=================  ===============  ============  ====================  ====================  ========
My Organization    org4da8bd3e334   a4da8bda70f5  2024-09-04T08:29:22Z  2024-09-04T08:29:22Z  business
```

```
avn account list
ACCOUNT_ID    ACCOUNT_NAME     ACCOUNT_OWNER_TEAM_ID  CREATE_TIME           IS_ACCOUNT_MEMBER  IS_ACCOUNT_OWNER  ORGANIZATION_ID  PARENT_ACCOUNT_ID  PRIMARY_BILLING_GROUP_ID  ROOT_ACCOUNT_ID  TENANT_ID  UPDATE_TIME         
============  ===============  =====================  ====================  =================  ================  ===============  =================  ========================  ===============  =========  ====================
a4da8bda70f5  My Organization  at4da8bd3236d          2024-09-04T08:29:22Z  true               true              org4da8bd3e334   null               null                      a4da8bda70f5     aiven      2024-09-04T08:29:22Z
```